contract-map.json

```diff
@@ -638,5 +638,75 @@
     "erc20": true,
     "symbol": "LIKE",
     "decimals": 18
+  },
+  "0xB8c77482e45F1F44dE1745F52C74426C631bDD52": {
+    "name": "Binance Coin",
+    "logo": "bnb.png",
+    "erc20": true,
+    "symbol": "BNB",
+    "decimals": 18
+  },
+  "0xd850942ef8811f2a866692a623011bde52a462c1": {
```

Comment:
This address and others are failing due to format, if you go to [etherscan](https://etherscan.io/address/0xd850942ef8811f2a866692a623011bde52a462c1) you will see `0xD850942eF8811f2A866692A623011bDE52a462C1` with capital letters.